pkg/helpers/reconcile.go

```diff
@@ -215,7 +215,17 @@ func (helper *ReconcileHelper) reconcileApplications() error {
 			return err
 		}
 
-		if appCh.Dependencies() != nil {
+		// if not subcharts (dependencies specified) then stage the application chart without disabling the subcharts
+		// to be deployed as one unit
+		var disableSubcharts bool
+
+		if application.Spec.Subcharts == nil || len(application.Spec.Subcharts) == 0 {
+			// staged application chart should have the subcharts enabled
+			disableSubcharts = false
```

Comment:
Bool is false by default, no need to set it

Reply:
Agree - this is mostly for being explicit that having nil or empty subcharts this bool is disabled. Benign in either case